lib/generators/active_fedora/noid/templates/migrations/create_minter_states.rb

```diff
@@ -0,0 +1,12 @@
+class CreateMinterStates < ActiveRecord::Migration
```

Comment:
I think you can put this in db/migrations.  Then you can use `rails g active_fedora-noid:install:migrations` rather than having to move them in a custom generator.